src/components/component.ts

```diff
@@ -282,8 +282,6 @@ export class Component {
     }
     this.computeLayout();
     this.render();
-    // flush so that consumers can immediately attach to stuff we create in the DOM
-    RenderController.flush();
```

Comment:
Apparently was added in PR #1082? I don't see a repro of the specific problem in the ticket (#1030) though.